engine/deployments/docker-compose/1m1e.yaml

```diff
@@ -20,15 +20,19 @@ services:
                         - "--advertise-peer-urls"
                         - "http://server-master:8291"
                         - "--frame-meta-endpoints"
-                        - "frame-mysql-standalone:3306"
+                        - "mysql-standalone:3306"
```

Comment:
Better to configure such parameters in config file.

Reply:
Fix it.